base/namedtuple.jl

```diff
@@ -44,6 +44,16 @@ julia> collect(pairs(x))
  :a => 1
  :b => 2
 ```
+
+A named tuple can be created programmatically by splatting an iterator yielding
+`name => value` pairs where each `name` is a `Symbol`:
+
+```jldoctest
+julia> keys = (:a, :b, :c); values = (1, 2, 3);
+
+julia> (; zip(keys, values)...)
+(a = 1, b = 2, c = 3)
+```
```

Comment:
:+1: While we're at it, can also mention the syntax `(; :a => 1)` (where the name is computed for one element at a time).

Reply:
Perhaps explicitly call out the commonality to calling functions with keyword arguments?

Reply:
Updated.